package/kernel/linux/modules/leds.mk

```diff
@@ -99,6 +99,22 @@ endef
 $(eval $(call KernelPackage,ledtrig-oneshot))
 
 
+define KernelPackage/ledtrig-pattern
+  SUBMENU:=$(LEDS_MENU)
+  TITLE:=LED Pattern Trigger
+  KCONFIG:=CONFIG_LEDS_TRIGGER_PATTERN
+  FILES:=$(LED_TRIGGER_DIR)/ledtrig-pattern.ko
+  AUTOLOAD:=$(call AutoLoad,50,ledtrig-pattern)
```

Comment:
Dos this also work with?
```
AUTOLOAD:=$(call AutoProbe,ledtrig-pattern)
```

Reply:
From what I understand about the code: sadly no.

LED triggers can be set through either the sysfs led "trigger" file (through led_trigger_write() )
or through device-tree (led_trigger_set_default() as part of led_classdev_register_ext() ) .

In both cases, the code will just iterate over a list of known/loaded triggers. 
If the trigger name is found, it will get set, if it isn't in the list, then it will either 
just skip it (in case of led_trigger_set_default()) or return -EINVAL (sysfs).

So, the trigger would need to be manually loaded. 

(Should it also be available at boot? If so, please add a ,1)